dll/win32/newdev/newdev.c

```diff
@@ -366,12 +371,20 @@ SearchDriverRecursive(
         ok = FindNextFileW(hFindFile, &wfd))
     {
 
+        /* Filename is too long to be searched */
+        if (wcslen(wfd.cFileName) + 1 > MAX_PATH)
```

Comment:
```suggestion
        if (wcslen(wfd.cFileName) >= _countof(FileName))
```

Reply:
Ok, done + include stdlib for definition of _countof